### PR TITLE
Update SPA template versions

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fixes aspnet/AspNetCore-Internal#2684.

Just running `npm audit fix` wasn't enough because there were conflicting versions of babel-eslint. In the end the only way I could figure out to both not have conflicting dependencies and not have any vulnerabilities was to totally remove `eslint-config-react-app`. @danroth27 for customer experience thoughts on that and @javiercn for engineering concerns it might raise.

### CODE REVIEW

**Comment by javiercn** [comment]
That will degrade the config experience for react. There must be a vulnerability free version you can install that doesn’t offer conflicts.

**Comment by ryanbrandenburg** [comment]
:up::date:

**Comment by javiercn** [comment]
Do we have any warnings left? It would be good to understand that

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/src/store/configureStore.ts
Comment (javiercn) [inline src/ProjectTemplates/Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/src/store/configureStore.ts]: Seems this got added to the review by mistake. Can you confirm this is on purpose and matches the expectations of other files in the template?

**Review by javiercn — APPROVED** [review]
Looks great! Is there any warning left? It would be good for the next time if you show the before/after so that we can understand what is left if anything.

Can you confirm the unrelated file change before merging?

**Comment by ryanbrandenburg** [comment]
audit

**Comment by ryanbrandenburg** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=238895

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2717

**Review by danroth27 — APPROVED** [review]

**Pull request merged**
